indico/modules/events/registration/client/styles/regform.module.scss

```diff
@@ -120,7 +120,43 @@
 }
 
 .section-actions {
-  a + a {
-    margin-left: 3px;
+  a {
+    font-size: 1.2em;
   }
+
+  a + a,
+  a + div {
+    margin-left: 5px;
+  }
+}
+
+.dropdown-menu {
+  display: inline-grid;
+  grid-template-columns: 1fr 1fr;
+
+  & > div:first-child {
+    border-right: 1px solid rgba(34, 36, 38, 0.15);
+  }
+}
+
+.dropdown-item {
+  cursor: pointer;
+  line-height: 1em;
+  color: rgba(0, 0, 0, 0.87);
+  padding: 0.78571429rem 1.14285714rem;
```

Comment:
yuck.. i guess those are calculated values from SUI? any way we can avoid replicating them here?

Reply:
yeah, the dropdown requires a very specific structure otherwise the css is not applied so I had to duplicate it. I'll try playing around with it a bit more..

Reply:
I at least replaced the padding with normal looking numbers but I think the rest will have to stay. The dropdown item styles are defined with the child selector (>) on the dropdown container but to get the two column layout I needed to insert some extra divs in between..